docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -1,7 +1,11 @@
 [[monitor-gcp]]
-== Monitor Google Cloud Platform
+= Monitor Google Cloud Platform
```

Comment:
Maybe add "(GCP)" to the end so both the full name and the acronym are in the title, which might have an influence on SEO.